catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -395,26 +404,9 @@ private void updateConfiguration(String metacardTitle) {
         try {
             Configuration currentConfig = configAdmin.getConfiguration(currentPid);
             Dictionary<String, Object> currentProperties = currentConfig.getProperties();
-            URI uri;
-            String registryUrl = (String) currentProperties.get(REGISTRY_URL);
-            try {
-                uri = new URI(registryUrl);
-            } catch (URISyntaxException e) {
-                LOGGER.debug("Unable to update registry configurations. Syntax error on registryUrl",
-                        e);
-                return;
-            }
-            String port = "";
-            if (uri.getPort() != noPortFound) {
-                port = ":" + uri.getPort();
-            }
-            String connectionType = getConnectionType(this.getBundleContext()
-                    .getBundle(), (String) currentProperties.get("service.factoryPid"));
-            String nameToSet = String.format("%s (%s%s) (%s)",
-                    metacardTitle,
-                    uri.getHost(),
-                    port,
-                    connectionType);
+            String nameToSet = getFullRegistryName(metacardTitle);
+            LOGGER.info("Changed RegistryStore.id from {} to {}", getId(), nameToSet);
```

Comment:
This will not occur very often. In practice this will most likely only happen once. Would like to keep it at info if that's ok.